src/components/header/SkinTonePicker.tsx

```diff
@@ -55,16 +59,52 @@ export function SkinTonePicker({
 
   const vertical = direction === SkinTonePickerDirection.VERTICAL;
 
+  const getHorizontalTranslation = ({
+    ix,
+    fanOutDirection,
+    isOpen,
+  }: {
+    ix: number;
+    fanOutDirection: SkinTonePickerFanOutDirection;
+    isOpen: boolean;
+  }): string => {
+    // By fanning out to the left, the focus remains on the last (right-most) tone in the array,
+    // so tabbing takes a user out of the SkinTonePicker. In order to tab through the tones, a user
+    // must first tab backwards.
+    //
+    // Fanning out to the right keeps the focus on the first (left-most) tone in the array so a user
+    // can tab from left to right.
+    if (fanOutDirection === SkinTonePickerFanOutDirection.LEFT) {
+      return `translateX(-${ix * (isOpen ? ITEM_SIZE : 0)}px)`;
+    }
+    return `translateX(${ix * (isOpen ? ITEM_SIZE : 0) -
+      (isOpen ? (skinToneVariations.length - 1) * ITEM_SIZE : 0)
+      }px)`;
+  };
+
+  const buttonStyle = { backgroundColor: "transparent", border: "none" }
+
   return (
     <Relative
       className={clsx('epr-skin-tones', direction, {
-        [ClassNames.open]: isOpen
+        [ClassNames.open]: isOpen,
       })}
       style={
         vertical
-          ? { flexBasis: expandedSize, height: expandedSize }
-          : { flexBasis: expandedSize }
+          ? { flexBasis: expandedSize, height: expandedSize, ...buttonStyle }
+          : { flexBasis: expandedSize, ...buttonStyle }
       }
+      button={true}
```

Comment:
This approach leads to non-standard HTML, each of the skintones is a button, so this leads to buttons nested under a button, which is not standard and is likely to break in different scenarios.
![image](https://user-images.githubusercontent.com/11255103/213886038-3167fc89-c7f6-4b0a-8499-40c560fc9e88.png)
![image](https://user-images.githubusercontent.com/11255103/213886042-2c013165-2c97-470c-8d22-826cf88fc5ae.png)

Reply:
Instead, what if tabbing would just trigger the same function hitting the right arrow does?
https://github.com/ealush/emoji-picker-react/blob/8e4cad3501cf25f618e424730f72248ba1520d59/src/hooks/useKeyboardNavigation.ts#L135